tests/test_kafka.py

```diff
@@ -0,0 +1,10 @@
+import kafka
+
+from testcontainers.kafka import KafkaContainer
+
+
+def test_docker_run_kafka():
+    with KafkaContainer() as container:
```

Comment:
Can we try this with at least two different ports to make sure they all line up?

Reply:
I would like to suggest to also test publishing and consuming messages.

Reply:
@tillahoffmann Could you please elaborate what do you mean by two different ports here?